### PR TITLE
Created problem_39 in project_euler

### PR DESCRIPTION
**Pull request by Kush1101**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by Kush1101** [comment]
@cclauss I need your assistance.

File: project_euler/problem_39/sol1.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    Returns a dictionary with keys as the perimeter of a right angled triangle
+    and value as the number of corresponding triplets.
+    >>> pythagorean_triple(15)
+    {12: 1}
+    >>> pythagorean_triple(40)
+    {12: 1, 30: 1, 24: 1, 40: 1, 36: 1}
+    >>> pythagorean_triple(50)
+    {12: 1, 30: 1, 24: 1, 40: 1, 36: 1, 48: 1}
+    """
+    triplets = {}
```
Comment (cclauss) [inline project_euler/problem_39/sol1.py:23]: Triplets should be a https://docs.python.org/3/library/collections.html#collections.Counter

Comment (Kush1101) [reply project_euler/problem_39/sol1.py:23]: Ok. I'll make the change

File: project_euler/problem_39/sol1.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                    if perimeter in triplets:
+                        triplets[perimeter] += 1
+                    else:
+                        triplets[perimeter] = 1
+    return triplets
+
+
+if __name__ == "__main__":
+    triplets = pythagorean_triple(1000)
+    print(max(triplets, key=lambda key: triplets[key]))
```
Comment (cclauss) [inline project_euler/problem_39/sol1.py:42]:
```suggestion
    print(f"{triplets.most_common()[0][0] = }")
```

File: project_euler/problem_39/sol1.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            hypotenuse = (base * base + perpendicular * perpendicular) ** 0.5
+            if hypotenuse == int((hypotenuse)):
+                perimeter = int(base + perpendicular + hypotenuse)
+                if perimeter > max_perimeter:
+                    continue
+                else:
+                    if perimeter in triplets:
+                        triplets[perimeter] += 1
+                    else:
+                        triplets[perimeter] = 1
```
Comment (cclauss) [inline project_euler/problem_39/sol1.py:36]: Replace lines 33-36 with just line 34.

Comment (Kush1101) [reply project_euler/problem_39/sol1.py:36]: Done.

**Comment by Kush1101** [comment]
Can it be merged now? @cclauss

**Comment by Kush1101** [comment]
@cclauss  sir, Any feedback will be helpful for me to improve my PR's.?

**Pull request merged**